packages/module-backup/src/client/Configuration.tsx

```
@@ -253,6 +271,7 @@ const NewBackup = ({ ButtonComponent = Button, refresh }) => {
   };
 
   const handleOk = (method) => {
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)